dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/DatadogGesturesTracker.kt

```diff
@@ -6,22 +6,25 @@
 
 package com.datadog.android.rum.internal.instrumentation.gestures
 
-import android.app.Activity
+import android.content.Context
+import android.view.View
+import android.view.Window
 import com.datadog.android.rum.tracking.ViewAttributesProvider
 import java.lang.ref.WeakReference
 
 internal class DatadogGesturesTracker(
     internal val targetAttributesProviders: Array<ViewAttributesProvider>
-) :
-    GesturesTracker {
+) : GesturesTracker {
 
     // region GesturesTracker
-    override fun startTracking(activity: Activity) {
+
+    override fun startTracking(window: Window?, context: Context) {
```

Comment:
now that we changed this I am thinking maybe we should not have an optional there for the Window and instead to put the Nullability condition before calling the function. If this is a big change you can leave it like this and open a ticket instead and I will take care of it next sprint.

Reply:
Never mind this.